diff -ur vegastrike.orig/src/Makefile.am vegastrike/src/Makefile.am
--- vegastrike.orig/src/Makefile.am	2004-07-15 01:12:26.000000000 +0200
+++ vegastrike/src/Makefile.am	2004-07-31 01:23:19.867251360 +0200
@@ -1,5 +1,5 @@
 
-bin_PROGRAMS = vegastrike vegaserver accountserver
+bin_PROGRAMS = vegastrike vegaserver
 
 SUBDIRS = \
     cmd gfx gldrv aldrv python gui networking boost common boost129
@@ -47,12 +47,6 @@
 boost/libboost_python.a	        \
 -lz
 
-accountserver_LDADD =           \
-networking/libnetacct.a 		\
-networking/lowlevel/libnetlowlevel.a \
--lz
-
-
 libroot =		\
 	debug_vs.h				\
 	debug_vs.cpp			\
@@ -152,25 +146,6 @@
 	$(librootgeneric) $(librootserver)	\
 	vegaserver.cpp
 
-accountserver_SOURCES =		\
-	vsfilesystem.cpp		\
-	vsfilesystem.h			\
-	pk3.cpp					\
-	pk3.h					\
-	posh.cpp				\
-	libserver.cpp			\
-	configxml.cpp			\
-	configxml.h				\
-	easydom.cpp				\
-	easydom.h				\
-	xml_support.h  			\
-	xml_support.cpp			\
-	lin_time.cpp			\
-	lin_time.h				\
-	endianness.cpp			\
-	endianness.h			\
-	accountserver.cpp
-	
 #	galaxy_gen.cpp
 #	galaxy_gen.h
 
