diff -urN vegastrike.orig/src/gldrv/gl_globals.h vegastrike/src/gldrv/gl_globals.h
--- vegastrike.orig/src/gldrv/gl_globals.h	2003-06-30 07:22:59.000000000 +0200
+++ vegastrike/src/gldrv/gl_globals.h	2004-04-29 23:34:29.609549016 +0200
@@ -20,6 +20,7 @@
  */
 #ifndef _GL_GLOBALS_H_
 #define _GL_GLOBALS_H_
+#define glXGetProcAddress glXGetProcAddressARB
 #include <queue>
 
 using namespace std;
diff -urN vegastrike.orig/src/gldrv/gl_init.h vegastrike/src/gldrv/gl_init.h
--- vegastrike.orig/src/gldrv/gl_init.h	2002-11-23 06:46:11.000000000 +0100
+++ vegastrike/src/gldrv/gl_init.h	2004-04-29 23:34:51.092283144 +0200
@@ -27,6 +27,7 @@
 #ifndef _GL_INIT_H_
 #define _GL_INIT_H_
 
+#define glXGetProcAddress glXGetProcAddressARB
   //#include "vegastrike.h"
 
 /* Hack to fix compiling problem with old gl.h's, reported by Steve
